shotover/src/transforms/kafka/sink_cluster.rs

```diff
@@ -55,7 +54,7 @@ pub struct KafkaSinkClusterBuilder {
     shotover_nodes: Vec<KafkaAddress>,
     connect_timeout: Duration,
     read_timeout: Option<Duration>,
-    coordinator_broker_id: Arc<AtomicI32>,
+    coordinator_broker_id: Arc<DashMap<StrBytes, i32>>,
```

Comment:
Why the change from an Atomic int to a regular one? Does Dashmap give what we need?

Reply:
An `Arc<Atomic<I32>>` is used instead of an `Arc<i32>` because `Arc<Atomic<I32>>` is read/write while `Arc<i32> is read only.
`Arc<DashMap<StrBytes, i32>` is already read/write, so making it an `Arc<DashMap<StrBytes, AtomicI32>` would be redundant.